src/app/ui/pdbe-viewport-controls.tsx

```diff
@@ -1,136 +1,230 @@
-import { PluginConfig } from 'Molstar/mol-plugin/config';
-import { ControlGroup } from 'Molstar/mol-plugin-ui/controls/common';
-import { ToggleSelectionModeButton } from 'Molstar/mol-plugin-ui/structure/selection';
-import { DownloadScreenshotControls } from './pdbe-screenshot-controls';
-import { SimpleSettingsControl } from 'Molstar/mol-plugin-ui/viewport/simple-settings';
-import { ViewportControls } from 'Molstar/mol-plugin-ui/viewport';
-import { AutorenewSvg, CameraOutlinedSvg, BuildOutlinedSvg, FullscreenSvg, TuneSvg, CloseSvg, BlurOnSvg } from 'Molstar/mol-plugin-ui/controls/icons';
-import { PluginUIComponent } from 'molstar/lib/mol-plugin-ui/base';
-import { VolumeStreamingControls, VolumeSourceControls } from 'molstar/lib/mol-plugin-ui/structure/volume';
+import { PluginConfig } from "Molstar/mol-plugin/config";
+import { ControlGroup } from "Molstar/mol-plugin-ui/controls/common";
+import { ToggleSelectionModeButton } from "Molstar/mol-plugin-ui/structure/selection";
+import { DownloadScreenshotControls } from "./pdbe-screenshot-controls";
+import { SimpleSettingsControl } from "Molstar/mol-plugin-ui/viewport/simple-settings";
+import { ViewportControls } from "Molstar/mol-plugin-ui/viewport";
+import {
+    AutorenewSvg,
+    CameraOutlinedSvg,
+    BuildOutlinedSvg,
+    FullscreenSvg,
+    TuneSvg,
+    CloseSvg,
+    BlurOnSvg,
+} from "Molstar/mol-plugin-ui/controls/icons";
+import { PluginUIComponent } from "molstar/lib/mol-plugin-ui/base";
+import {
+    VolumeStreamingControls,
+    VolumeSourceControls,
+} from "molstar/lib/mol-plugin-ui/structure/volume";
```

Comment:
When working with forks, we must (unfortunately) follow the indentation of that project, whatever that is. Otherwise, merging with upstream changes will be a nightmare. So here we should see only our real changes, but no prettification. Probably you'll need to disable "Format on save" on vscode for that workspace.